app/src/main/java/com/smouldering_durtles/wk/views/SubjectInfoButtonView.java

```diff
@@ -24,6 +24,7 @@
 import android.graphics.Color;
 import android.graphics.ColorFilter;
 import android.graphics.Paint;
+import android.graphics.Path;
```

Comment:
Path was added in https://github.com/jerryhcooke/smouldering_durtles/commit/437fbab760c74f98fe2972e08dbf3f7833139340 but without the import. Added it so this would build; not sure how it built otherwise. 🤔